Jenkinsfile

```
@@ -0,0 +1 @@
+common {}
```

```suggestion
common {
  cron = null
}
```

We don't need this being triggered on a cron schedule.